Cpix/Constants.cs

```diff
@@ -44,12 +44,12 @@ static class Constants
 		/// 
 		/// Values are name-namespace pairs.
 		/// </summary>
-		public static readonly Tuple<string, string>[] TopLevelXmlElementOrder = new Tuple<string, string>[]
+		public static readonly Tuple<string, string>[] TopLevelXmlElementOrder =
```

Comment:
I can't highlight the exact line for some reason, but at line 41 "oder" should be "order".